src/components/CartPage/CartPage.js

```diff
@@ -97,7 +102,13 @@ const CartPage = (props) => {
             <p>{`$${deliveryCost}`}</p>
             <p>{`$${voucher}`}</p>
             <hr />
-            <p>{`$${countTotal + deliveryCost - voucher}`}</p>
+            <input
+              className="total-price"
+              value={`$${countTotal + deliveryCost - voucher}`}
+              type="text"
+              onChange={getPrice(+`${countTotal + deliveryCost - voucher}`)}
```

Comment:
Does it work ?

Reply:
it works, but i'll replace - as you wish)